src/snowflake/connector/feature.py

```diff
@@ -3,7 +3,17 @@
 #
 # Copyright (c) 2012-2021 Snowflake Computing Inc. All right reserved.
 #
+import os
 
 # Feature flags
 
 feature_use_pyopenssl = True  # use pyopenssl API or openssl command
+
+
+# use the new code-path without using SDK to put and get files
+def feature_sdkless_put() -> bool:
+    return os.environ.get("SF_SDKLESS_PUT", "").lower() == "true"
+
+
+def feature_sdkless_get() -> bool:
+    return os.environ.get("SF_SDKLESS_GET", "").lower() == "true"
```

Comment:
Why is this needed?

Reply:
Because we are not removing the old code paths right away, so these environmental variables will switch to the new code paths separately for PUTs and GETs